src/Engine/babylon.engine.ts

```diff
@@ -5205,6 +5205,9 @@
         }
 
         public setTextureFromPostProcessOutput(channel: number, postProcess: Nullable<PostProcess>): void {
+            if(postProcess._outputTexture._disposed){

```

Comment:
this could be late to the party
I would prefer doing it in the internalTexture dispose to do it asap